### DISCUSSION TITLE
Saving changes while working with workspace database is slow

### DISCUSSION THREAD

**Discussion by panikkos13** [TabularEditor/TabularEditor3#933] [🙋♀️TE3 Q&A] [answered]
Hi there community,

Currently running the 3.8.0.18402 version Business Edition.

Saving changes on workspace database is quite slow when the database contains data. 

We use PowerBI premium workspace to host the workspace database, also tried compatibility level 1600. When we clear data of the workspace database, saving changes takes much less time.

Also, each developer uses their own PBI workspace and workspace database.

Anything we can do to make saving changes faster while also maintaining the data?

**Comment by otykier** [comment, marked as answer]
Have you enabled large datasets on the Power BI workspace?

> While required for datasets to grow beyond 10 GB, enabling the Large dataset storage format setting has other benefits. If you're planning to use XMLA endpoint-based tools for dataset write operations, be sure to enable the setting, even for datasets that you wouldn't necessarily characterize as a large dataset. **When enabled, the large dataset storage format can improve XMLA write operations performance.**

https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-large-models

**Reply by panikkos13** [reply]
While we had the large datasets enabled to our production models, we didn't enable it to our workspace database models. 

Enabling this solve the issue. Thank you very much!
